app/src/test/java/nerd/tuxmobil/fahrplan/congress/navigation/RoomForC3NavConverterTest.kt

```diff
@@ -0,0 +1,41 @@
+package nerd.tuxmobil.fahrplan.congress.navigation
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import org.junit.runners.Parameterized.Parameters
+
+@RunWith(Parameterized::class)
+class RoomForC3NavConverterTest(
+
+        private val room: String?,
+        private val expectedText: String
+
+) {
+
+    companion object {
```

Comment:
shouldn't it be `private companion` instead of ` private fun scenarioOf`?

Reply:
:x: The `data` function below has to marked as `@JvmStatic`. This annotation is not allowed in a `private` `companion` object.